src/apprt/gtk/Window.zig

```diff
@@ -284,7 +284,7 @@ pub fn init(self: *Window, app: *App) !void {
         const toolbar_view: *c.AdwToolbarView = @ptrCast(c.adw_toolbar_view_new());
 
         const header_widget: *c.GtkWidget = self.header.?.asWidget();
-        c.adw_toolbar_view_add_top_bar(toolbar_view, header_widget);
```

Comment:
This seems like an unrelated change that should not be part of this PR.

Reply:
Fixed